packages/ove-app-networks/README.md

```diff
@@ -0,0 +1,57 @@
+# Networks App
+
+This app supports visualisation of network diagrams using the OVE framework. It is based on [Sigma](http://sigmajs.org/), a JavaScript library dedicated to drawing graphs. This supports datasets specified in Sigma's JSON format or [GEXF](https://gephi.org/gexf/format/). Sigma supports Canvas, SVG and WebGL rendering.
```

Comment:
"This app provides support for the visualization of networks using node-link diagrams within the OVE framework"